eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -62,36 +62,17 @@ case class NodeParams(nodeKeyManager: NodeKeyManager,
                       private val overrideFeatures: Map[PublicKey, Features],
                       syncWhitelist: Set[PublicKey],
                       pluginParams: Seq[PluginParams],
-                      dustLimit: Satoshi,
-                      maxRemoteDustLimit: Satoshi,
+                      channelConf: ChannelConf,
                       onChainFeeConf: OnChainFeeConf,
-                      maxHtlcValueInFlightMsat: UInt64,
-                      maxAcceptedHtlcs: Int,
-                      expiryDelta: CltvExpiryDelta,
-                      fulfillSafetyBeforeTimeout: CltvExpiryDelta,
-                      minFinalExpiryDelta: CltvExpiryDelta,
-                      maxBlockProcessingDelay: FiniteDuration,
-                      maxTxPublishRetryDelay: FiniteDuration,
-                      htlcMinimum: MilliSatoshi,
-                      toRemoteDelay: CltvExpiryDelta,
-                      maxToLocalDelay: CltvExpiryDelta,
-                      minDepthBlocks: Int,
                       relayParams: RelayParams,
-                      reserveToFundingRatio: Double,
-                      maxReserveToFundingRatio: Double,
-                      unhandledExceptionStrategy: UnhandledExceptionStrategy,
                       db: Databases,
-                      revocationTimeout: FiniteDuration,
                       autoReconnect: Boolean,
                       initialRandomReconnectDelay: FiniteDuration,
                       maxReconnectInterval: FiniteDuration,
                       chainHash: ByteVector32,
-                      channelFlags: ChannelFlags,
                       watchSpentWindow: FiniteDuration,
```

Comment:
Shouldn't we move this one to `RouterConf`?

Reply:
Done with 827eda5.